common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -74,23 +74,38 @@ class @Problem
   renderProgressState: =>
     detail = @el.data('progress_detail')
     status = @el.data('progress_status')
+    graded = @el.data('graded')
 
     # Render 'x/y point(s)' if student has attempted question 
     if status != 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
         a = detail.split('/')
         earned = parseFloat(a[0])
         possible = parseFloat(a[1])
-        # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
-        `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
-        progress_template = ngettext('(%(earned)s/%(possible)s point)', '(%(earned)s/%(possible)s points)', possible)
+
+        if graded == "True" and possible != 0
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (graded)', '%(earned)s/%(possible)s points earned (graded)', possible)
+        else
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (ungraded)', '%(earned)s/%(possible)s points earned (ungraded)', possible)
         progress = interpolate(progress_template, {'earned': earned, 'possible': possible}, true)
 
     # Render 'x point(s) possible' if student has not yet attempted question
-    if status == 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
-        a = detail.split('/')
-        possible = parseFloat(a[1])
-        `// Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10). There will always be at least 1 point possible.`
-        progress_template = ngettext("(%(num_points)s point possible)", "(%(num_points)s points possible)", possible)
+    if status == 'none' or status == 0
+        if detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
```

Comment:
Looking more at this, I think it's a bit messier to fix it than it is to let it go. I can add in 0/0 rather than 0, but adding in the part that tells you that you have gotten 0 things correct after already started the problem requires adding something into the POST request that will change the values of details and progress. I'm a little hesitant to change a lot about the progress object and the API calls because I'm not sure how widely they're used.

Right now, details and progress are just being represented by one "variable" (the progress object that is "#/#" or "none") and there's no good way to represent the state of being in progress other than the way it's already done (which is to check if the first number is greater than 0). The progress object won't accept 0 for a max value, and won't accept -1 for a points earned value. Without changing the progress object, or setting the status in an outside method that is called after a POST request, there's no way to account for an extra state.

I can show you how I attempted to fix it tomorrow, if it'll make it any clearer.